modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -871,12 +871,12 @@ export class Tileset3D {
     this.stats.get(TILES_IN_MEMORY).incrementCount();
 
     // Good enough? Just use the raw binary ArrayBuffer's byte length.
-    this.gpuMemoryUsageInBytes += tile.content.byteLength || 0;
+    this.gpuMemoryUsageInBytes += tile.gpuMemoryUsageInBytes || 0;
```

Comment:
You could update tile.byteLength calculation in 3d-tiles module instead of changing it here

Reply:
Except it isn't the byte length of the tile it is the GPU memory used. Tile3D already has the `gpuMemoryUsageInBytes` field it was just always 0. We even have a comment here asking whether just inspecting the byte length is good enough, and it generally isn't because of the decompressed texture memory being larger than raw bytes

Reply:
In that case, could you move `gpuMemoryUsage` calculation to 3d-tiles module? glTF dependency is not good as `tile` module is abstract by design. Feel free to set new property like `content.gpuMemoryUsageInBytes` as `byteLength` is not correct